src/dawn/IIR/DoMethod.cpp

```diff
@@ -85,6 +90,91 @@ const std::shared_ptr<DependencyGraphAccesses>& DoMethod::getDependencyGraph() c
   return derivedInfo_.dependencyGraph_;
 }
 
+void DoMethod::DerivedInfo::clear() {
+  fields_.clear();
+}
+
+void DoMethod::clearDerivedInfo() { derivedInfo_.clear(); }
```

Comment:
should this not be the default implementation?

Reply:
do you mean the implementation of the base class?
I can not access the member of the derived from the base class, isnt?